interface.js

```diff
@@ -0,0 +1,36 @@
+view.addEventListener('click', function(){
+    showPeeps();
+})
+
+search.addEventListener('click', function(){
+    let element = document.getElementById("peep").value;
+    console.log(element)
+    singlePeep(element);
+})
+
+function showPeeps(){
+    fetch('https://chitter-backend-api-v2.herokuapp.com/peeps').then(function(res){
+    return res.json()
+}).then((data) => {
+    let text = [`<ul>`]
```

Comment:
Rather than building out the html as a string elements. You could perhaps look at interfacing with the DOM directly by using the `document.createElement()` syntax in combination with `appendChild()` syntax.

I.e. something along the lines of:

`let list = document.createElement('ul')`
`let listPeep = document.createElement('li')`
`listPeep.innerText = peep.body`
`list.appendChild(listPeep`

Reply:
This is great help , i wanted to implement this but didn't really know how too